packages/nc-gui/composables/useMultiSelect/convertCellData.ts

```
@@ -21,343 +17,128 @@
+    /**
+     * This method can throw errors. so it's important to use a try-catch block when calling it.
+     */
+    serializedValue = ColumnHelper.serializeValue(value, {
+      col: column,
+      isMysql: (_sourceId) => isMysql,
+      isMultipleCellPaste: isMultiple,
+    })
+
+    serializedValue = handlePostSerialize(serializedValue, value)
+  } catch (ex) {
+    throw ex
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)